root/components/ReleaseGroupsList.js

```diff
@@ -0,0 +1,263 @@
+  +noRatings?: boolean,
+  +releaseGroup: ReleaseGroupT,
+  +seriesItemNumbers?: {+[number]: string},
+  +showArtists?: boolean,
+|};
+
+type ReleaseGroupsListProps = {|
+  +$c: CatalystContextT,
+  +checkboxes?: string,
+  +noGrouping?: boolean,
+  +noRatings?: boolean,
+  +order?: string,
+  +releaseGroups: $ReadOnlyArray<ReleaseGroupT>,
+  +seriesItemNumbers?: {+[number]: string},
+  +showArtists?: boolean,
+  +sortable?: boolean,
+|};
+
+const ReleaseGroupsListHeader = ({
+  $c,
+  checkboxes,
+  noGrouping,
+  noRatings,
+  order,
+  seriesItemNumbers,
+  showArtists,
+  sortable,
+}: ReleaseGroupsListHeaderProps) => (
+  <thead>
+    <tr>
+      {$c.user_exists && checkboxes ? (
+        <th style={{width: '1em'}}>
+          <input type="checkbox" />
+        </th>
+      ) : null}
+      {seriesItemNumbers ? <th style={{width: '1em'}}>{l('#')}</th> : null}
+      <th className="year c">
+        {sortable
+          ? (
+            <SortableTableHeader
+              label={l('Year')}
+              name="year"
+              order={order}
+            />
+          )
+          : l('Year')}
+      </th>
+      <th>
+        {sortable
+          ? (
+            <SortableTableHeader
+              label={l('Title')}
+              name="name"
+              order={order}
+            />
+          )
+          : l('Title')}
+      </th>
+      {showArtists ? <th className="artist">{l('Artist')}</th> : null}
+      {noGrouping ? (
+        <th>
+          {sortable
+            ? (
+              <SortableTableHeader
+                label={l('Type')}
+                name="primary_type"
+                order={order}
+              />
+            )
+            : l('Type')}
+        </th>
+      ) : null}
+      {noRatings ? null : <th className="rating c">{l('Rating')}</th>}
+      <th className="count c">{l('Releases')}</th>
+    </tr>
+  </thead>
+);
+
+const ReleaseGroupsListEntry = ({
+  $c,
+  checkboxes,
+  index,
+  noGrouping,
+  noRatings,
+  releaseGroup,
+  seriesItemNumbers,
+  showArtists,
+}: ReleaseGroupsListEntryProps) => (
+  <tr className={loopParity(index)} key={releaseGroup.id}>
+    {$c.user_exists && checkboxes ? (
+      <td>
+        <input
+          name={checkboxes}
+          type="checkbox"
+          value={releaseGroup.id}
+        />
+      </td>
+    ) : null}
+    {seriesItemNumbers ? (
+      <td style={{width: '1em'}}>
+        {seriesItemNumbers[releaseGroup.id]}
+      </td>
+    ) : null}
+    <td className="c">
+      {releaseGroup.firstReleaseDate
+        ? new Date(releaseGroup.firstReleaseDate).getFullYear()
+        : '—'}
+    </td>
+    <td>
+      <EntityLink entity={releaseGroup} />
+    </td>
+    {showArtists ? (
+      <td>
+        <ArtistCreditLink
+          artistCredit={releaseGroup.artistCredit}
+        />
+      </td>
+    ) : null}
+    {noGrouping ? (
+      <td>
+        {releaseGroup.typeName
+          // TODO: replace with releaseGroupType when it works here
```

Comment:
utility/releaseGroupType currently uses $c.linked_entities. That does *not* contain RG type info for RG collections. @yvanzo said that $c.linked_entities is mostly intended for release pages - if that's the case we might want to consider switching how releaseGroupType works.